rootfs/Dockerfile

```diff
@@ -56,7 +58,7 @@ RUN curl -sSL https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.p
 
 # install deis/wal-e from the tip of the boto3-upgrade branch
 # NOTE (bacongobbler): we use the commit here so Docker's cache will bust when we make changes upstream
-RUN pip install --disable-pip-version-check --no-cache-dir git+https://github.com/deis/wal-e.git@dfa35c648b1a7b554122312d6c6b592be372e17a
+RUN pip install --disable-pip-version-check --no-cache-dir git+https://github.com/kmala/wal-e.git@3c1729302db6a38ae32709da24d91a5caa264499
```

Comment:
note to self: let's make sure to revert this once https://github.com/deis/wal-e/pull/4 is merged.